src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRequest.cs

```diff
@@ -204,5 +204,7 @@ public LockFile ExistingLockFile
         public bool RestoreForceEvaluate { get; set; }
 
         public IReadOnlyList<IAssetsLogMessage> AdditionalMessages { get; set; }
+
```

Comment:
Maybe add comment about when we set true. For now, it's only for `globalpackages` folder, right?